pkg/controllers/state/informer/node.go

```diff
@@ -0,0 +1,73 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package informer
```

Comment:
Why push these down a layer? For naming? Maybe easier to just make it `nodecontroller.go`

Reply:
Logically, too. There's the actual model and the things that inform the model. I liked the prospect of separating the informers and the actual data.